esmvalcore/cmor/check.py

```diff
@@ -283,12 +298,28 @@ def _check_dim_names(self):
                 except iris.exceptions.CoordinateNotFoundError:
                     try:
                         coord = self._cube.coord(coordinate.standard_name)
-                        self.report_error(
-                            'Coordinate {0} has var name {1} instead of {2}',
-                            coordinate.name,
-                            coord.var_name,
-                            coordinate.out_name,
-                        )
+                        if self._cmor_var.table_type in 'CMIP6' and \
+                           coord.ndim > 1 and \
+                           coord.standard_name in ['latitude', 'longitude']:
+                            self.report_debug_message(
+                                'Multidimensional {0} coordinate is not set '
+                                'in CMOR standard. ESMValTool will change '
+                                'the original value of  {1} to {2} to match '
+                                'the one-dimensional case.',
+                                coordinate.standard_name,
+                                coord.var_name,
+                                coordinate.out_name,
+                                )
+                            coord.var_name = coordinate.out_name
+                        else:
+                            self.report_error(
+                                'Coordinate {0} has var name {1}'
+                                'Coordinate {0} has var name {1} '
+                                'instead of {2}',
+                                coordinate.name,
+                                coord.var_name,
+                                coordinate.out_name,
+                                )
```

Comment:
codacy complaining about the indentation for the `)` - not sure why, maybe you have some blank space or smth?